Documentation/kubernetes.md

```diff
@@ -19,6 +19,8 @@ To make sure you have a Kubernetes cluster that is ready for `Rook`, you can [fo
 
 Note that we are striving for even more smooth integration with Kubernetes in the future such that `Rook` will work out of the box with any Kubernetes cluster.
 
+If you are using `dataDirHostPath` to persists rook data on kubernetes hosts, make sure your host have at least 5GB of space available on the specified path.
```

Comment:
looks great thanks! just a couple of typos if you could update to: `persist` (instead of `persists`) and `has` instead of `have`)